InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/auto/AutoConfig.java

```diff
@@ -0,0 +1,11 @@
+package frc.robot.subsystem.drive.auto;
+
+public class AutoConfig {
+    public static final double cruiseSpeed_mps = 0.3 * 14;
```

Comment:
If these values are configurable, they should regular class variables. We only want constants to be static final variables. 

These were all configurable values before.